psql2bigquery/main.py

```diff
@@ -97,14 +98,25 @@ def dump_table(cls, table_name: str):
 
     @classmethod
     def list_tables(cls):
```

Comment:
About this method, it looks like it's repeating the same for-loop twice. Can we move that to a separated function for a better reuse?

Reply:
Sure, I refactored it.